medicines/doc-index-updater/src/service_bus_client/mod.rs

```diff
@@ -0,0 +1,49 @@
+use azure_sdk_core::errors::AzureError;
+use azure_sdk_service_bus::prelude::*;
+use std::time::Duration;
+use tokio::time::delay_for;
+
+pub async fn delete_queue_client() -> Result<Client, AzureError> {
+    let service_bus_namespace = std::env::var("SERVICE_BUS_NAMESPACE")
+        .expect("Set env variable SERVICE_BUS_NAMESPACE first!");
+
+    let queue_name =
+        std::env::var("DELETE_QUEUE_NAME").expect("Set env variable DELETE_QUEUE_NAME first!");
+
+    let policy_name = std::env::var("DELETE_QUEUE_POLICY_NAME")
+        .expect("Set env variable DELETE_QUEUE_POLICY_NAME first!");
+
+    let policy_key = std::env::var("DELETE_QUEUE_POLICY_KEY")
+        .expect("Set env variable DELETE_QUEUE_POLICY_KEY first!");
+
+    Client::new(service_bus_namespace, queue_name, policy_name, policy_key)
+}
+
+pub async fn create_queue_client() -> Result<Client, AzureError> {
+    let service_bus_namespace = std::env::var("SERVICE_BUS_NAMESPACE")
+        .expect("Set env variable SERVICE_BUS_NAMESPACE first!");
+
+    let queue_name =
+        std::env::var("CREATE_QUEUE_NAME").expect("Set env variable CREATE_QUEUE_NAME first!");
+
+    let policy_name = std::env::var("CREATE_QUEUE_POLICY_NAME")
+        .expect("Set env variable CREATE_QUEUE_POLICY_NAME first!");
+
+    let policy_key = std::env::var("CREATE_QUEUE_POLICY_KEY")
+        .expect("Set env variable CREATE_QUEUE_POLICY_KEY first!");
+
+    Client::new(service_bus_namespace, queue_name, policy_name, policy_key)
+}
+
+pub async fn delete_service_worker() -> Result<String, AzureError> {
```

Comment:
Is the intention to move this worker to the delete service module?

Reply:
Indeed